src/components/overview/sub-Components/AddToBag.jsx

```diff
@@ -1,7 +1,56 @@
 import React from 'react';
 
+let sizeTypes = ['XS', 'S', 'M', 'L', 'XL'];
+let orderNumbers = [
+  1,
+  2,
+  3,
+  4,
+  5,
+  6,
+  7,
+  8,
+  9,
+  10,
+  11,
+  12,
+  13,
+  14,
+  15,
+  16,
+  17,
+  18,
+  19,
+  20
+];
+
 const AddToBag = function() {
-  return <div></div>;
+  return (
+    <div>
+      <div>
+        <div class="select is-medium">
+          <select>
+            <option>choose size</option>
+            {sizeTypes.map(eachSize => {
```

Comment:
These are really descriptive variable names. Makes understanding what is happening much simpler.